android/src/main/java/com/revenuecat/purchases/hybridcommon/mappers/StoreProductMapper.kt

```diff
@@ -36,12 +38,13 @@ fun StoreProduct.map(): Map<String, Any?> =
         "price" to priceAmountMicros / 1_000_000.0,
         "priceString" to priceString,
         "currencyCode" to priceCurrencyCode,
-        // TOOD: BC5 allows up to 2 intro phases (free and then discount), should this only show the first?
         "introPrice" to mapIntroPrice(),
         "discounts" to null,
         "productCategory" to mapProductCategory(),
         "productType" to mapProductType(),
-        "subscriptionPeriod" to period?.iso8601
+        "subscriptionPeriod" to period?.iso8601,
+        "defaultOption" to defaultOption?.mapSubscriptionOption(this),
+        "subscriptionOptions" to subscriptionOptions?.map { it.mapSubscriptionOption(this) },
```

Comment:
These new fields were needed to expose subscription options that can be purchased

Reply:
we'll need to add presentedOfferingId once my changes are merged

Reply:
Ah, sounds good! I'll do that in a separate PR since that requires a new Android version to be released but I'll create a JIRA task for that 💪